models/events/webhook/github/GithubCommit.js

```diff
@@ -0,0 +1,15 @@
+module.exports = require('../../../Model').extend({
```

Comment:
Shouldn't this have a typeName and an entry in EventType, or is that future work (maybe that's what you meant by 'bonus change?')?